fe/fe-core/src/main/java/org/apache/doris/qe/MasterOpExecutor.java

```diff
@@ -63,6 +63,9 @@ public void execute() throws Exception {
     private void forward() throws Exception {
         String masterHost = ctx.getCatalog().getMasterIp();
         int masterRpcPort = ctx.getCatalog().getMasterRpcPort();
+        if (masterHost == "" || masterRpcPort == 0) {
```

Comment:
Maybe this is better. Put it at the beginning.
```
if (!ctx.getCatalog().isReady()) {
    thrown new Exception("Node catalog is not ready, please wait for a while");
}
```

Reply:
> Maybe this is better. Put it at the beginning.
> 
> ```
> if (!ctx.getCatalog().isReady()) {
>     thrown new Exception("Node catalog is not ready, please wait for a while");
> }
> ```
agree